gateway/src/apicast/policy/3scale_referrer/3scale_referrer.lua

```diff
@@ -0,0 +1,20 @@
+local policy = require('apicast.policy')
+local _M = policy.new('3scale Referrer policy')
+
+local new = _M.new
+
+function _M.new(config)
+  local self = new(config)
+  return self
+end
+
+function _M.rewrite(_, context)
```

Comment:
It'd be nice to be able to use `export` here but can't because we need to add to the existing table, not overwrite it.